packages/core/src/utils/validators.ts

```diff
@@ -0,0 +1,153 @@
+export type ValidationValue = File | FormData | string | undefined;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type ValidationMessageCallback = (instance: any, value: ValidationValue) => string;
```

Comment:
Can we use `unknown` here instead of `any`?

Reply:
That will break all the locations where `instance` is passed.